playbooks/roles/sanity-checker/files/sanity_check.py

```diff
@@ -86,28 +87,36 @@ def send_message(recipient, subject, body):
 
 def get_dev_mounts():
     """
-    :return: List of mounted drives, we assume that drives will have
```

Comment:
Are these changes related to the pull request?

Reply:
@toxinu yes, please see https://github.com/open-craft/ansible-playbooks/pull/228/commits/0aaf9faf1553e3fee9f8cf8304c10fdb7031661f

It was previously broken on mail.opencraft.com because it was trying to get fs info on /var/lib/lxcfs